client/hudwidget.cpp

```diff
@@ -1834,17 +1837,24 @@ void hud_unit_combat::paintEvent(QPaintEvent *event)
   p.setFont(f);
   p.setPen(QColor(Qt::white));
   if (def_veteran) {
-    p.drawText(right,
-               Qt::AlignHCenter | Qt::AlignJustify | Qt::AlignAbsolute,
-               QStringLiteral("*"));
+    p.drawText(right, Qt::AlignBottom | Qt::AlignRight | Qt::AlignAbsolute,
+               _("*Vet*"));
   }
   if (att_veteran) {
-    p.drawText(left, Qt::AlignHCenter | Qt::AlignJustify | Qt::AlignAbsolute,
-               QStringLiteral("*"));
+    p.drawText(left, Qt::AlignBottom | Qt::AlignRight | Qt::AlignAbsolute,
+               _("*Vet*"));
   }
   p.drawText(left, Qt::AlignHorizontal_Mask, ahploss);
   p.drawImage(right, dimg);
   p.drawText(right, Qt::AlignHorizontal_Mask, dhploss);
+  if (def_win) {
+    p.drawText(right, Qt::AlignBottom | Qt::AlignLeft | Qt::AlignAbsolute,
+               _("Win"));
```

Comment:
I suggest `Winner` here and/or leave a note for translators (`Win`/`Gagne` would make no sense, we really want `Victor`/`Vainqueur` in French).

Reply:
Or maybe `Wins`...

Reply:
I'll change to "Winner" and add a //TRANS